src/modules/public.ts

```diff
@@ -230,6 +231,23 @@ export default class Public {
     );
   }
 
+  /**
+   * @description get retroactive mining rewards for an ethereum address
+   *
+   * @param ethereumAddress An Ethereum address
+   */
+  getPublicRetroactiveMiningRewards(
+    ethereumAddress: string,
+  ): Promise<PublicRetroactiveMiningRewardsResponseObject> {
```

Comment:
up for debate but I think it's actually _easier_ for users if you have the whole object in the promise rather than creating an additional type for like a 2 field object that they then need to check and might mess up

Reply:
mentioned offline: for promises, i think keeping the format of return types consistent would be nice